boards/arm/nrf54h20pdk_nrf54h20/Kconfig.board

```diff
@@ -0,0 +1,10 @@
+# Copyright (c) 2024 Nordic Semiconductor ASA
```

Comment:
@anangl Please let me know when RST files will be in the PR

Reply:
> @anangl Please let me know when RST files will be in the PR

@FrancescoSer RST file is now present in the PR.